.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--
+🚨HOLD UP PLEASE 🚨
```

Comment:
I'm a little worried that it scares of users who have never opened an issue on GitHub 🙈

Reply:
It sais please :( I'm scared about people not reading it... What would you suggest?